_maps/map_files/generic/Centcomm.dmm.rej

```diff
@@ -0,0 +1,11 @@
+diff a/_maps/map_files/generic/Centcomm.dmm b/_maps/map_files/generic/Centcomm.dmm	(rejected hunks)
+@@ -12455,8 +12455,7 @@
+ /turf/closed/wall/mineral/titanium/overspace,
```

Comment:
Github's text editor is starting to lag a little bit down here.

One more reject for the pile.